src/firebase-adapter.ts

```diff
@@ -181,6 +184,8 @@ export class FirebaseAdapter implements IDatabaseAdapter {
   }
 
   dispose(): void {
+    this._zombie = true;
```

Comment:
I think this should be added after `if (!this._ready)` condition.

Or add a cleanup `this._zombie = false` before return

Reply:
closing, discussed on call